web/styles/compose.css

```diff
@@ -1058,7 +1058,8 @@ textarea.new_message_textarea {
     .enter_sends_choice {
         display: flex;
         gap: 8px;
-        padding: 4px 10px;
+        margin: 0 3.2px;
```

Comment:
Why 3.2px? I don't see why a fractional pixel would do anything.

Reply:
Yes, you are right that a fractional pixel won't do anything. I was converting rem into pixels, which is why I introduced the fractional pixel there. I am rounding it to 3px.